apps/web-client/src/lib/db/cr-sqlite/order-reconciliation.ts

```diff
@@ -204,6 +204,50 @@ export async function finalizeReconciliationOrder(db: DB, id: number) {
 	});
 }
 
+/**
+ * Retrieves all supplier orders that have not yet been included in any reconciliation process.
+ * @param db
+ *
+ * @returns {Promise<SupplierPlacedOrderLine[]>} Array of unreconciled supplier orders with:
+ *  - id: The supplier order ID
+ *  - supplier_id: The ID of the supplier
+ *  - supplier_name: The name of the supplier
+ *  - created: Creation timestamp of the order
+ *  - total_book_number: Total quantity of books in the order (0 if no books)
```

Comment:
```suggestion
```

Removes the description of the `SupplierPlacedOrderLine` properties. This was Silvio's major feedback on my jsdoc PR